updaters/github.js

```diff
@@ -0,0 +1,173 @@
+                  for (var file of filesJson.tree) {
+                    if (file.path.indexOf(cBasePath) === 0) {
+                      if (minimatch(file.path, cRule, {
+                        nodir: true,
+                        realpath: true
+                      })) {
+                        newFiles.push({path: file.path, tree: fullData.tree})
+                      }
+                    }
+                  }
+                  files = files.concat(newFiles);
+                  if (newFiles.length === 0) {
+                    console.log('Not found'.red, cRule.cyan, tag);
+                    fs.mkdirsSync(path.normalize(path.join(__dirname, '../../cdnjs', 'ajax', 'libs', library.name, tag)));
+                  }
+                });
+                allFiles = allFiles.concat(files.map(function (c) {
+                  return {
+                    _: c.path,
+                    basePath: cBasePath,
+                    tree: c.tree
+                  }
+                }));
+              });
+              console.log('All files for ' + library.name + ' v' + tag, '-', allFiles.length);
+              library.version = library.version || "0.0.0";
+              var greaterVer;
+              try {
+                greaterVer = compareVersions(tag, library.version) > 0;
+              } catch (e) {
+                greaterVer = false;
+              }
+              if (
+                  (allFiles.length !== 0) &&
+                  (
+                   (!library.version) ||
+                   (
+                    (greaterVer) &&
+                    (
+                     (stable.is(tag)) ||
+                     (!stable.is(tag) && !stable.is(library.version))
+                    )
+                   )
+                  )
+                 ) {
+                console.log('Updated package.json to version'.green, tag);
+                var libraryPath = path.normalize(path.join(__dirname, '../../cdnjs', 'ajax', 'libs', library.name, 'package.json'));
+                var libraryJSON = JSON.parse(fs.readFileSync(libraryPath, 'utf8'));
+                libraryJSON.version = tag;
+                fs.writeFileSync(libraryPath, JSON.stringify(libraryJSON, undefined, 2) + '\n');
+              }
+              async.each(allFiles, function(file, callback) {
+                var fileName = file._;
+                var fileTarget = path.normalize(path.join(__dirname, '../../cdnjs', 'ajax', 'libs', library.name, tag, fileName));
+                fs.ensureFile(fileTarget, function(err) {
+                  if (err) {
+                    console.log('Some strange error occured here'.red);
+                    console.dir(err);
+                    callback();
+                  } else {
+                    rp({
+                      uri:'https://raw.githubusercontent.com/'
+                        + githubRepo(library.autoupdate.target) + '/'
+                        + file.tree + '/' + file.basePath + '/' + fileName,
```

Comment:
Could change this to `path.normalize(path.join(file.tree, file.basePath, fileName)),`

This is a path for a website, but I think the benefit still applies.

Reply:
Sounds good!